### PR TITLE
feat:adds the signup and login page

### PR DESCRIPTION
**Pull request by FarhanMobashir**
-add the basic signup and login card

### CODE REVIEW

File: pages/authentication.html [hunk 1]
```diff
@@ -4,10 +4,149 @@
...
+            <a class="navigation-link" href="#">WOMEN</a>
+            <a class="navigation-link" href="#">KIDS</a>
+            <a class="navigation-link" href="#">OFFERS</a>
+          </div>
+          <div class="search-container-big">
+            <i class="search-icon uil uil-search"></i>
+            <input
+              placeholder="Search here..."
+              class="search-input-big"
+              type="text"
```
Comment (MaruthiPothuganti) [inline pages/authentication.html:37]: you can use type="search" for searchbar

File: pages/authentication.html [hunk 2]
```diff
@@ -4,10 +4,149 @@
...
+              <i class="badge-icon uil uil-user-circle"></i>
+            </a>
+          </div>
+        </div>
+        <div class="search-container-small">
+          <i class="search-icon uil uil-search"></i>
+          <input
+            placeholder="Search products here..."
+            class="search-input-small"
+            type="text"
```
Comment (MaruthiPothuganti) [inline pages/authentication.html:64]: same as above: type="search"

**Review by MaruthiPothuganti — COMMENTED** [review]
Suggested a change. Clean code and Everything else looks Good.

**Review by Shivanshsharma01 — APPROVED** [review]
The code looks clean and well structured. Can go ahead with the merge

**Pull request merged**